hammerblade/torch/hb_pytorch_link.ld

```diff
@@ -0,0 +1,184 @@
+/******************************/
```

Comment:
This looks very important. Could you maybe lecture us what is difference between dram and dmem?

Reply:
`dmem` is the scratch pad and `dram` is the main memory of HB. Debug messages like printing and constants are stored in `rodata` section, which by default goes into `dmem` in Manycore's link script. We would want those to go to `dram` as scratchpad is a scarce resource.

Reply:
ahh I see.